soroban-env-common/env.json

```diff
@@ -1373,52 +1373,52 @@
                             "type": "StorageType"
                         },
                         {
-                            "name": "low_expiration_watermark",
+                            "name": "threshold",
                             "type": "U32Val"
                         },
                         {
-                            "name": "high_expiration_watermark",
+                            "name": "extend_to",
                             "type": "U32Val"
                         }
                     ],
                     "return": "Void",
-                    "docs": "If the entry expiration is below `low_expiration_watermark` ledgers from the current ledger (inclusive), then bump the expiration to be `high_expiration_watermark` from the current ledger (inclusive)"
+                    "docs": "If the entry expiration is below `threshold` ledgers from the current ledger (inclusive), then bump the expiration to be `extend_to` from the current ledger (inclusive)"
```

Comment:
Remove the term "expiration" and "bump" from the docs. Maybe something like "If the entry's TTL is below `threshold` ledgers, extend `liveUntilLedger` such that `TTL == extend_to`, where TTL is defined as `liveUntilLedger - currentLedger`. I think we should definitely expose the term TTL in the SDK, but this doc may be too low level for the term, I'm not sure.

Reply:
Fixed